spring-cloud-huawei-config/src/main/java/com/huaweicloud/config/client/ServiceCombConfigClientBuilder.java

```diff
@@ -45,14 +47,19 @@ public ServiceCombConfigClientBuilder setServiceCombSSLProperties(
     return this;
   }
 
-  public ServiceCombConfigClientBuilder setUrl(String url) {
-    this.url = url;
+  public ServiceCombConfigClientBuilder setServiceCombConfigProperties(
+      ServiceCombConfigProperties serviceCombConfigProperties) {
+    this.serviceCombConfigProperties = serviceCombConfigProperties;
     return this;
   }
 
   public ServiceCombConfigClient createServiceCombConfigClient() {
     DefaultHttpTransport httpTransport = DefaultHttpTransport.getInstance(serviceCombSSLProperties);
     httpTransport.setServiceCombAkSkProperties(serviceCombAkSkProperties);
-    return new ServiceCombConfigClient(url, httpTransport);
+    if (!StringUtils.isEmpty(serviceCombConfigProperties.getServerType())
+        && serviceCombConfigProperties.getServerType().equals("kie")) {
```

Comment:
You can use "kie".equalsIgnoreCase to make logic simpler.

Reply:
very useful